src/framework/uicomponents/qml/MuseScore/UiComponents/StyledPopupView.qml

```diff
@@ -39,7 +39,7 @@ PopupView {
     property int padding: 16
     property int margins: 16
 
-    property int contentWidth: contentBody.childrenRect.width
+    property int contentWidth: 240
```

Comment:
Curious about this change, why this value is strictly defined now

Reply:
Previously, I set this value in each popup, that is, it was duplicated in many places.
Now I decided to move it to the base class so as not to duplicate it.